src/app.js

```diff
@@ -13,15 +13,23 @@ const { deleteCatByIndex } = require('./catsDb.js');
 const isInvalidString = (value) => {
     if (typeof value === "string") {
         return false;
-    } else if (typeof value === "undefined") {
+    } else {
         return false;                     
     };                                   
 }
 // if value is undefined, then we can let it be ""; 
 
+const isInvalidSex = (value) => {
+    if (value !== 'M' || 'F') {
```

Comment:
consider allowing an 'other' value here. there's a slight bug in the right-most condition also, as it's evaluating the truthyness of `'F'` which will always be true